### PR TITLE
fix(helm-charts): add support for new format of appVersion fields

### PR DESCRIPTION
**Pull request by vponomaryov**
Latest Helm versions wrap values of "appVersion" field into quotes
to make yaml parser consider it always as string to avoid ambiguity.
Example 1: 1.0 may be considered as float
Example 2: 1234e10 as scientific notation.

Details: https://helm.sh/docs/topics/charts/#the-appversion-field

And operator-1.6-rc.0 started using Helm version that does it.
So, add support for it in our "get_operator_image" method which
we use in our K8S functional tests.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/cluster_k8s/__init__.py [hunk 1]
```diff
@@ -488,7 +488,8 @@ def get_operator_image(self, repo: str = '', chart_version: str = '') -> str:
             f"show chart {chart_name} --devel --repo {repo} --version {chart_version}")
         for line in chart_info.split("\n"):
             if line.startswith("appVersion:"):
-                return f"scylladb/scylla-operator:{line.split(':')[-1].strip()}"
+                found_app_version = line.split(':')[-1].strip().replace('"', '').replace("'", "")
```
Comment (dkropachev) [inline sdcm/cluster_k8s/__init__.py:491]: Could you please add comment with example appVersion we are parsing here

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: sdcm/cluster_k8s/__init__.py (hunk 1 above)
Comment (vponomaryov) [reply sdcm/cluster_k8s/__init__.py:504]: Done

**Review by dkropachev — APPROVED** [review]

File: sdcm/cluster_k8s/__init__.py (hunk 1 above)
Comment (enaydanov) [reply sdcm/cluster_k8s/__init__.py:504]: What do you think about changing it to 
```python
found_app_version =line.removeprefix("apiVersion:").strip(" \t\"'")
```
?

Comment (vponomaryov) [reply sdcm/cluster_k8s/__init__.py:504]: I don't see any need to change it. Moreover, I get following result using proposed code:

```
appVersion: "1.6
```

and the existing code in PR provides proper results:
```
1.6
```

Comment (enaydanov) [reply sdcm/cluster_k8s/__init__.py:504]: Typo in the suggested code: "apiVersion:" -> "appVersion:" Sorry about that.

**Review by enaydanov — APPROVED** [review]

**Pull request merged**